ami/ml/post_processing/tests/test_small_size_filter_admin.py

```
@@ -0,0 +1,120 @@
+
+from ami.jobs.models import Job
+from ami.main.models import SourceImageCollection
+from ami.tests.fixtures.main import setup_test_project
+from ami.users.models import User
+
+
+class _SmallSizeFilterAdminCase(TestCase):
+    def setUp(self) -> None:
+        self.superuser = User.objects.create_superuser(
+            email="ssfadmin@example.com",
+            password="x",
```

Ruff (0.15.15)

[error] Possible hardcoded password assigned to argument: "password"

(S106)